src/latexdocvisitor.cpp

```diff
@@ -1284,6 +1284,10 @@ void LatexDocVisitor::visitPre(DocHtmlCell *c)
   {
     m_t << "\\cellcolor{\\tableheadbgcolor}\\textbf{ ";
   }
+  else 
+  {
+    if (c->isFirst()) m_t << "\\hspace{0pt}";
+  }
```

Comment:
When not mistaken I saw this proposed patch also as part of #8025. Probably it is better to have as a separate proposed patch, but in that case it should be removed from #8025

Reply:
You are correct I meant #8005 (got confused by nearly the same numbers).